srcpkgs/applet-window-appmenu/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'applet-window-appmenu'
+pkgname=applet-window-appmenu
+version=0.6.0
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release  .. -Wnodev"
+make_build_args="-j$(nproc)"
+hostmakedepends="extra-cmake-modules gcc"
```

Comment:
`gcc` is unneded.

Reply:
Adding `qt5-host-tools` and `qt5-qmake` to `hostmakedepends` should get past the cross build issue you are experiencing. These two packages are the packages that seem to be missing.

This works on native because it is pulled in in makedepends too.